backend/app/services/article_vector_service.py

```diff
@@ -46,29 +46,8 @@ def __init__(self):
         )
 
     def create_collection_safe(self, collection_name: str) -> None:
```

Comment:
```suggestion
    def create_collection(self, collection_name: str) -> None:
```

I don't think we need this here - it was for me when I was working in this file to understand the difference between the two functions (one catches errors while the other can fail). but since you removed one now, it's ok without the `_safe` I think